build-tools/create-pkg/distribution.xml.in

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<installer-gui-script minSpecVersion="1">
+    <license file="License" mime-type="application/rtf" />
+    <title>@PACKAGE_TITLE@</title>
+    <pkg-ref id="xamarin.android-oss">
+        <bundle-version/>
+    </pkg-ref>
+    <choices-outline>
+        <line choice="default">
+            <line choice="xamarin.android-oss"/>
+        </line>
+    </choices-outline>
+    <choice id="default"/>
+    <choice id="xamarin.android-oss" visible="false">
+        <pkg-ref id="xamarin.android-oss"/>
+    </choice>
+    <pkg-ref id="xamarin.android-oss">#xamarin.android-oss.pkg</pkg-ref>
```

Comment:
Does this need to be `xamarin.android-oss` and not just `xamarin.android`?  Remember that the eventual goal is that we use these targets to produce the eventual commercial `.pkg` file, so unless we allow this value to be overridden via some `@FOO@`...

Reply:
I was thinking that we could specify the path to the `distribution.xml.in` file vs an MSBuild property that will default to the file in this repo, but can be overridden by other invocations via `/p:PkgDistributionSrcFile` or something of the like. Though if you would prefer to use the same file and produce OSS and commercial packages under the same name, that can be done instead.

Reply:
At present, I don't see what changing the name would actually gain.

Reply:
I've updated the distribution and pkgutil / productbuild calls to remove the unnecessary `-oss` prefixes. I've also added the property `$(PkgNamePrefix)` to control the .pkg file name. This defaults to  `xamarin.android-oss` and can be overridden.